packages/cli/src/commands/lint.ts

```diff
@@ -16,12 +16,19 @@
 
 import { Command } from 'commander';
 import { run } from '../lib/run';
+import { paths } from '../lib/paths';
 
 export default async (cmd: Command) => {
-  const args = ['lint', '--max-warnings=0', '--format=codeframe'];
+  const args = [
+    '--ext',
+    'js,jsx,ts,tsx',
```

Comment:
did this one not support the one-word `=` joined version? :)

Reply:
Copied from web-scripts, so didn't try it :grin: